package.json

```diff
@@ -34,7 +34,7 @@
     "nx-build-test-skip-cache": "pnpm run build-test --skip-nx-cache",
     "nx-test-skip-cache": "pnpm run test --skip-nx-cache",
     "build": "nx run-many -t build --parallel=1",
-    "build-test": "nx run-many -t style,build,test",
+    "build-test": "nx run-many -t style,build,test --projects=@openzeppelin/defender-base-client,*  --parallel=false",
```

Comment:
This is the relevant change, the rest are just pre-commit format changes

Reply:
This should not be necessary since parallel=1 should still run sequentially. Is it not the case?

Reply:
Doesn't seem to work :/ it tries to run `notification-channel:build` before `base:build`.

By the docs it seems tha the way to run secuentially is by setting `parallel=false`

![Screenshot 2023-10-05 at 15 08 17](https://github.com/OpenZeppelin/defender-sdk/assets/14266096/de1d18d1-6578-42aa-8b58-4abebd1460af)

Reply:
Interesting. Sounds good. Thanks